src/components/Slider/RangeSlider/RangeSlider.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import {
+  Root as SliderRoot,
+  Track as SliderTrack,
+  Range as SliderRange,
+  Thumb as SliderThumb
+} from '@radix-ui/react-slider';
+
+import styles from '../Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+export interface RangeSliderProps {
+  values?: number[];
+  min: number;
+  max: number;
+  step: number;
+  minStep?: number;
+  isSmall?: boolean;
+  isLight?: boolean;
+}
+
+export const RangeSlider: React.FC<RangeSliderProps> = ({ step, min, max, values, isSmall, minStep, isLight }) => {
+  const [currentValues, setCurrentValues] = useState(values ?? [min, max]);
+  return (
+    <div className={cx(styles.RootContainer, { LightRootContainer: isLight })}>
+      {currentValues[0]} - {currentValues[1]}
+      <form>
```

Comment:
The documentation says that its so the events from the input thumbs are propagated properly. https://www.radix-ui.com/docs/primitives/components/slider

Reply:
I see - makes sense. I think you can remove the `<form>` here, I think the documentation is saying that it will change the thumb elements to `input`s if rendered in a `form`, and outside of a `form` they'll be something else (probably `div`).

Reply:
ohh right, Ill remove it then